src/stdatamodels/jwst/datamodels/tests/test_schema_against_crds.py

```diff
@@ -163,6 +163,7 @@ def flatten(xs):
     'sirskernel': dm.SIRSKernelModel,
     'speckernel': dm.SpecKernelModel,
     'specprofile': dm.SpecProfileModel,
+    'specpsf': dm.SpecPsfModel,
```

Comment:
```suggestion
    'psf': dm.SpecPsfModel,
```

The reftype is `psf` on crds:
https://jwst-crds.stsci.edu/browse/jwst_miri_psf_0002.rmap
Is this being renamed to `specpsf`?